### ISSUE TITLE
BLE/ANT+ sensor values not shown in Bicycle mode

### ISSUE THREAD

**Issue by euphi** [osmandapp/OsmAnd#18217]
### Description

Values from BLE sensors are not shown in the bicycle mode. Instead, a "-" is shown.

They are ok in other modes (World, Car, ..)

### Steps to reproduce

1. Configure at least one BLE or ANT+ sensor.
2. Add it to some display configuration
3. In bicycle mode, only the - value is shown.

--> changing to World or Car mode shows the sensor value. Switch back to Bicycle shows "-" again.

I tried to find differences between modes (or what is special in *my* bicyvle mode and tried:

1. 2D/3D mode
2. Bright/Dark mode
3. various locations of the sensor values
4. transparent widgets on/off.

The problem persists. I also tried to set World and Bicyle mode as similar as possible, there is no difference.

### Actual result

see steps to reproduce

### Expected result

the BLE/ant+ sensors value should be shown whenever they are configured to be shown.

### Your Environment (required)

```
OsmAnd Version: Osmand~ 4.5.10
Android/iOS version: 12
Device model: realme 7 5G
```

**Comment by euphi** [comment]
I'll upload some screenshots soon (add  to this comment)

**Comment by DmitryAlexei** [comment]
Is your sensor visible in menu > settings > Cycling (app profiles) > External Sensors (Plugin settings)?

**Comment by euphi** [comment]
Yes, it is shown (in alle profiles, so also in the profiles that do not show the heart rate value). 

If I open the sensor settings, it is shown that sensor is connected and I can see the value (+ value of batterie service).

**Comment by DmitryAlexei** [comment]
OsmAnd~ 4.6.0#866m, released: 2023-10-12
In some profiles sensors are not activated in widget *by default*. So when you add *hart rate* widget, for example, to bicycle profile, you need to activate sensor in widget manually (see screenshots):

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/64fb1e23-b9e6-40f9-ac90-95f027e61982"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/73be1290-8c20-4e51-9d8c-41268d3ae520"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by euphi** [comment]
Well... thanks for the hint. Still a strange behavior that should be fixed.

Either have no default in all profiles or use the same default in all.

A good solution would be to directly add the sensor instance as widget to the panel. (There is no need to have an "abstract" heart rate as sensor).

**Comment by euphi** [comment]
My preferred solution would be that a BLE widget by default shows the first currently connected sensor of this type.

Rationale: If you have more than one sensor of a type, you usually don't use them at the same time. However, there is no need to have two different profiles if you use two different bicylces (with individual speed and cadence sensors each).

**Comment by vshcherb** [comment]
Looks related to https://github.com/osmandapp/OsmAnd/issues/17936.

**Comment by vshcherb** [comment]
Unfortunately automatic selection is only fixed last week, so you need latest release. 
Well, indeed to have specific setting option - by default shows the first currently connected sensor of this type.

**To do**: add option "any available sensor of this type". For recording this option should be present but not selected by default

**Comment by DmitryAlexei** [comment]
OsmAnd~ 4.7.0#1346m, released: 2024-01-10

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/7382f5ee-d60f-4bf1-ba26-5e03b8128f18"  width="300" height="auto"/> </td>
  </tr>
</table>
